packages/core/mesh/teleport/src/control-extension.ts

```diff
@@ -106,17 +109,20 @@ export class ControlExtension implements TeleportExtension {
                 remotePeerId: this.remotePeerId.truncate(),
               });
             } else {
-              log('heartbeat RTT', {
-                rtt: now - resp.requestTimestamp.getTime(),
-                localPeerId: this.localPeerId.truncate(),
-                remotePeerId: this.remotePeerId.truncate(),
-              });
+              if (DEBUG_PRINT_HEARTBEAT) {
+                log('heartbeat RTT', {
+                  rtt: now - resp.requestTimestamp.getTime(),
+                  localPeerId: this.localPeerId.truncate(),
+                  remotePeerId: this.remotePeerId.truncate(),
+                });
+              }
             }
           }
         } catch (err: any) {
           const now = Date.now();
           if (err instanceof RpcClosedError) {
             log('ignoring RpcClosedError in heartbeat');
+            this._extensionContext.close(err);
```

Comment:
speed up rpc failure detection

Reply:
so this covers the case where a remote peer closes the RPC first.
Perhaps a better way would be to add a `close` event to RpcPeer and propagate the closure to the extension context